include/sot/core/device.hh

```diff
@@ -93,6 +93,7 @@ namespace dynamicgraph {
       void setVelocitySize(const unsigned int& size);
       virtual void setVelocity(const dg::Vector & vel);
       virtual void setSecondOrderIntegration();
+      virtual void setNoIntegration();
```

Comment:
if `setNoIntegration` is only needed in the `AbstractSotExternalInterface`, then why did you add it to `Device`?